src/test/java/hw2/ex1/Exercise1.java

```diff
@@ -0,0 +1,43 @@
+package hw2.ex1;
+
+import hw2.base.BaseTest;
+import org.testng.annotations.Test;
+
+public class Exercise1 extends BaseTest {
+
+    @Test
+    public void testingExercise_1() {
+        //1.Open test site by URL
+        openURL();
+
+        //2.Assert Browser title
+        assertTitle();
+
+        //3.Perform login
+        performLogin();
+
+        //4.Assert Username is loggined
+        assertUserName();
+
+        //5.Assert that there are 4 items on the header section are displayed and they have proper texts
+        assertItemsOnTheHeaderSection();
```

Comment:
Let's imagine that you have 2 header with same structure (in header of age and in left side).
They has same structure (ul, li and etc.) BUT they have different id-s: header-section-1, header-section-2.

But to assert both of them you need to use 1 method...got problem? :)
Method should be smth like:

` assertItemsOnTheHeaderSection(List<String> assertionData, List<WebElement> checkingElements);`